crates/ruff_linter/src/rules/flake8_pytest_style/snapshots/ruff_linter__rules__flake8_pytest_style__tests__PT028.snap

```diff
@@ -0,0 +1,165 @@
+---
+source: crates/ruff_linter/src/rules/flake8_pytest_style/mod.rs
+---
+PT028.py:3:16: PT028 [*] Test function parameter `a` has default argument
```

Comment:
Let's add some tests similar to the ones in the upstream implementation that verify the correct behavior for different parameter kinds.